swift/hummingbird-framework/config.yaml

```diff
@@ -0,0 +1,8 @@
+framework:
+  website: https://github.com/hummingbird-project
+  version: 0.6.0
+  name: hummingbird
```

Comment:
is this already require @0xTim ? I mean to have a directory not named `hummingbird`

Reply:
It's a limitation of the swift package manager. It gets confused if you have a dependency with the same name as the folder you are in. It thinks there is a cyclical dependency.